src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -204,6 +207,74 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
             }
         }
+
+        [RavenAction("/studio/whats-new", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetWhatsNewData()
+        {
+            if (ServerVersion.Build == ServerVersion.DevBuildNumber)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                return;
+            }
+
+            var license = ServerStore.LoadLicense();
+
+            if (license == null)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
```

Comment:
This is not a bad request. License it was not even provided to the request. Please note that RavenDB can run without a license (AGPL applies then) so we just need to ignore that